jwala-persistence/src/main/java/com/cerner/jwala/persistence/jpa/domain/resource/config/template/JpaGroupAppConfigTemplate.java

```diff
@@ -20,6 +20,9 @@
         @NamedQuery(name = JpaGroupAppConfigTemplate.GET_GROUP_APP_TEMPLATE_META_DATA,
```

Comment:
Remove this query and replace instances where this is used with the new query. We have to remove this query because it violates the model constraints now.